### ISSUE TITLE
Reload

### ISSUE THREAD

**Issue by colormyapp** [uacaps/PageMenu#30]
Hi there,

I was wondering, could I replace the 

self.navigationItem.rightBarButtonItem = UIBarButtonItem(title: "->", style: UIBarButtonItemStyle.Done, target: self, action: "didTapGoToRight")

with a button, that will reload the Controller? On the controller, I'm having UIWebView and loading content from URL. 

Thanks!

**Comment by fahlout** [comment]
Yes. you can put whatever UIBarButton you want to use in your application on either side of the navigation bar. The go right/left buttons were just for demo purposes

**Comment by TangSirOnGit** [comment]
HI, how to update var controllerArray : [UIViewController] content?

because in my app, the data is blank, I have to update the controllerArray after I got data from server.

thanks a lot.

**Comment by fahlout** [comment]
As it is written currently you will probably have to re-instantiate a new instance of page menu with your new array holding the controllers
